transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -239,10 +239,14 @@
   ;;     "{\"platforms\":\"file://frozen/platforms.csv\"}"
   (let [gcmd-resource-name (keyword-scheme->kms-resource keyword-scheme)]
     (if (str/starts-with? gcmd-resource-name "file://")
-      (->> (str/replace gcmd-resource-name #"^file://" "")
+      (do
+        (warn (format "Loading static KMS resource from %s for %s."
```

Comment:
I think this `warn` as well as the one on line 258 would make more sense as (`info`...)